2-library/reactive-postgres-persistent/src/main/resources/application-postgres-library.properties

```diff
@@ -0,0 +1,8 @@
+spring.r2dbc.url=
+spring.r2dbc.username=
+spring.r2dbc.password=
+spring.r2dbc.pool.max-size=10
+spring.r2dbc.pool.initial-size=5
+spring.r2dbc.pool.max-idle-time=30m
+spring.r2dbc.pool.max-life-time=1h
+spring.main.lazy-initialization=true
```

Comment:
we need to have lazy init only for `database`, 
This property will make all `spring beans` lazy. 

This logic is not the same. 

Pls,create a new ticket `Add lazy initialization for postgres library` 
and for now lets forgot about this. 
```suggestion
```